src/main/java/core/basesyntax/Rectangle1.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Rectangle1 extends Figure {
```

Comment:
```suggestion
public class Rectangle extends Figure {
```

Это плохая практика использовать цифры в названиях классов и премеенных